Backend/src/database/models/project.ts

```diff
@@ -1,63 +1,51 @@
 import {Document, model, Schema} from "mongoose";
 
-export enum visibilityStatus {
-    "PRIVATE",
-    "PUBLIC",
-    "BY_INSTITUTION",
+export enum ProjectStatus {
+    "UPLOAD_PENDING",
+    "UPLOAD_COMPLETE",
+    "PROCESSING_PENDING",
+    "ABORTED",
+    "DONE"
 }
+
 export interface IProject extends Document {
-   title:string,
-   description:string,
-   adminIds: Array<Schema.Types.ObjectId>,
-   invitedMemberIds: Array<Schema.Types.ObjectId>,
-   memberIds: Array<Schema.Types.ObjectId>,
-   visibility: string,
-   institutionId: Schema.Types.ObjectId,
+    owner: Schema.Types.ObjectId;
+
+    // file
+    uploadId: string;
+    location: string;
+    fileName: string;
+    fileSize: number; // (of bytes)
+    uploadDate: Date;
+
+    // team
```

Comment:
```suggestion
    // project
```
Still saves information about the project status